test/test_smtlib.cpp

```diff
@@ -372,3 +373,147 @@ TEST(TestSMTLib, TestBitVectorArithmeticXor) {
 
     ASSERT_EQ(ss.str(), output);
 }
+
+TEST(TestSMTLib, TestModelInt) {
+    using namespace logicbase;

```

Comment:
if you are using it in every test, you might as well just put a single `using namespace logicbase` at the top.